prr/services/providers/elevenlabs/tts.py

```diff
@@ -0,0 +1,27 @@
+from elevenlabs import generate, set_api_key
```

Comment:
there's `elevenlabs` package missing in pyproject.toml,
you can add it with `poetry add elevenlabs`